src/Teams/Controllers/ManageTeamsController.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Teams.Models;
+
+namespace Teams.Controllers
+{
+    public class ManageTeamController : Controller
```

Comment:
Please add some default public methods with the views to be able to server requests from the client